package.json

```diff
@@ -44,7 +44,12 @@
     },
     "./dist/*": {
       "import": "./dist/*"
-    }
+    },
+    "./dist/plugins/*.esm.js": {
+      "import": "./dist/plugins/*.esm.js",
+      "types": "./dist/plugins/*.d.ts",
+      "require": "./dist/plugins/*.cjs"
+    },
```

Comment:
JSON is unfortunately strict about trailing commas.

Reply:
haha oops. I used the GitHub editor for the quick PR, and just copy/pasted from my VSCode.

Reply:
08571f3b4a9d68a9537aec7fef984d9afd4d8b86